locales/fi.schema.json

```diff
@@ -610,7 +610,7 @@
             "label": "Muotokuva"
           },
           "options__3": {
-            "label": "Neliö"
+            "label": "Square"
```

Comment:
I think this is still not supposed to be that way. Shouldn't be the english word for it.